### ISSUE TITLE
Improved navigation between products / versions

### ISSUE THREAD

**Issue by vandan** [MetaMask/metamask-docs#700]
@eriknson commented on [Tue Apr 04 2023](https://github.com/MetaMask/mm-docs-v2/issues/97)

Let's add a filter for (1) product and (2) version above the left-side-nav instead of our current tabs in the top bar to:
- Visually connect the filtering with the left-side-nav.
- Create a surface where we can briefly introduce each of our products via the subtitles.
- Support versioning.
- Scale smoothly if we want to add more top-level filters.

<img width="1651" alt="Docs navigation-closeup" src="https://user-images.githubusercontent.com/34306844/229746300-e715a5fe-49e6-4693-a55f-c8beb9a3d4f2.png">

**Overview**
<img width="1651" alt="Docs-navigation" src="https://user-images.githubusercontent.com/34306844/229732846-33fe18bf-4ec9-4ed3-9b55-d87f8ed34c68.png">

---

@vandan commented on [Tue Apr 04 2023](https://github.com/MetaMask/mm-docs-v2/issues/97#issuecomment-1496248836)

The Product Filter + Version looks nice. It keeps the top-nav clean. 

In this arrangement, do you have thoughts about where we would put an intro or general concepts?
I'm thinking we could still default to Wallet API, which could include general guidelines and prominent references to the other products.

---

@eriknson commented on [Wed Apr 05 2023](https://github.com/MetaMask/mm-docs-v2/issues/97#issuecomment-1498139129)

@vandan exactly, I'm thinking we would have a default choice which would include that kind of content

**Comment by andrewjcohen** [comment]
I've been digging into docusaurus documentation, looking at local installs, etc and can not find anything suggesting that it supports this functionality.

**Comment by vandan** [comment]
Right, it's not out-of-the-box functionality, but I believe it's possible to customize in case we believe it's significantly impactful.

Some references for a site that does use dropdowns in a docusaurus project:
https://docs.moralis.io/web3-data-api/evm/reference
https://github.com/MoralisWeb3/docs

May be accomplished through Swizzling:
https://docusaurus.io/docs/swizzling
